### PR TITLE
protect multi set add data, use default bd anf fs for tests

### PR DESCRIPTION
**Pull request by offirko**
KVStore Integration tests branch
    [X] Fix
    [ ] Refactor
    [ ] Target update
    [ ] Functionality change
    [ ] Docs update
    [ ] Test update
    [ ] Breaking change

### CODE REVIEW

**Comment by offirko** [comment]
@theamirocohen , @dannybenor , @davidsaada , @yossi2le - New PR with set_add_data updates and FileSystemStore Test uses default BD and FS

File: features/storage/TESTS/kvstore/filesystemstore/fsst_tests/main.cpp [hunk 1]
```diff
@@ -30,17 +30,13 @@
 using namespace utest::v1;
 using namespace mbed;
 
-//SPIFBlockDevice bd(PTE2, PTE4, PTE1, PTE5);
-SPIFBlockDevice bd(MBED_CONF_SPIF_DRIVER_SPI_MOSI, MBED_CONF_SPIF_DRIVER_SPI_MISO, MBED_CONF_SPIF_DRIVER_SPI_CLK,
-                   MBED_CONF_SPIF_DRIVER_SPI_CS);
+BlockDevice *bd = BlockDevice::get_default_instance();
```
Comment (dannybenor) [inline features/storage/TESTS/kvstore/filesystemstore/fsst_tests/main.cpp:33]: You need to be prepared for targets where there are no components, therefore you will not get a bd from get_default_instance();

**Review by dannybenor — CHANGES_REQUESTED** [review]

File: features/storage/TESTS/kvstore/filesystemstore/fsst_tests/main.cpp (hunk 1 above)
Comment (offirko) [reply features/storage/TESTS/kvstore/filesystemstore/fsst_tests/main.cpp:33]: Thanks, should I skip the test if it returns NULL, or is there an additional option to define a BlockDevice with a FS and test such a target ?

Comment (offirko) [reply features/storage/TESTS/kvstore/filesystemstore/fsst_tests/main.cpp:33]: @dannybenor - is there an additional option to define a BlockDevice with a FS and test such a target ?

**Pull request closed without merging**